sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -361,7 +366,7 @@ typedef struct az_iot_hub_client_c2d_request
  * @param[out] out_request If the message is a C2D request, this will contain the
  *                         #az_iot_hub_client_c2d_request
  * @return #az_result
- *         - #AZ_ERROR_IOT_TOPIC_NO_MATCH if the topic is not matching the expected format.
+ *         - `AZ_ERROR_IOT_TOPIC_NO_MATCH` if the topic is not matching the expected format.

```

Comment:
Why was this change necessary? It should link to the correct `az_result` value with `#`

Reply:
Unfortunately it doesn't and leaves `#AZ_ERROR_IOT_TOPIC_NO_MATCH` in the doc.

Reply:
I think that is because that error code isn't documented. I use `#AZ_OK` and others in `az_span.h` etc. and it works fine, because it has a `///<` comment.